heal-remedy.lic

```diff
@@ -0,0 +1,124 @@
+# quiet
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#heal-remedy
+=end
+
+custom_require.call(%w[common common-healing common-items])
+
+
+arg_definitions = [
+  [
+    { name: 'debug', regex: /debug/i, optional: true }
+  ]
+]
+
+args = parse_args(arg_definitions)
+$debug_mode_hr = args.debug
+
+$remedies = get_data('remedies')
+
+def wounds
+  wounds = DRCH.check_health
+  pause 1
+  echo (wounds)  if $debug_mode_hr
+  hurt = wounds.select{|level, _| level >= 4}.values.flatten
+  wound_grouping = { "right arm" => "limbs", "left arm" => "limbs", "left leg" => "limbs", "right leg" => "limbs", "right hand" => "limbs", "left hand" => "limbs", "right eye" => "eyes", "left eye" => "eyes" }
+  scar_grouping = { "right arm" => "limbs", "left arm" => "limbs", "left leg" => "limbs", "right leg" => "limbs", "right hand" => "limbs", "left hand" => "limbs", "head" => "face", "neck" => "face", "eyes" => "face", "chest" => "body", "abdomen" => "body", "back" => "body" }
+  hurt = hurt.map{ |raw| wound_grouping[raw] || raw}.uniq
+  scar = hurt.map{ |raw| scar_grouping[raw] || raw}.uniq
+  echo ("Wounds: ") if $debug_mode_hr
```

Comment:
You could move this into the conditional below, instead of having two conditionals

Reply:
Which part are you referring? The echo or the groupings?

Reply:
the echo statement could be added below line 30

Reply:
The echo is the header for the list of wounds.

Reply:
I understand, just pointing out that you dont need to do `if $debug_mode_hr` twice, if the header is below line 30

Reply:
if $debug_mode_hr
    echo ("Wounds: ")
    hurt.each do |key| 
      puts key
    end
  end

Done.